### ISSUE TITLE
Electron Printing  extra white page only on horizontal pdfs

### ISSUE THREAD

**Issue by sgil-carecloud** [react-pdf-viewer/react-pdf-viewer#287]
I wanted to start a thread on this just to see if anyone else has been seeing this before, or where in the lib can actually start looking into this

**Comment by phuoc-ng** [comment]
Thanks for reporting the bug. Does this happen with vertical view mode?

**Comment by sgil-carecloud** [comment]
@phuoc-ng hey, thank you for the fast reply!, ok so this happens when i hit the print button, click on save as pdf, if the pdf is a horizontal pdf, it prints an extra page. The preview is perfect, and up to the print, everything looks good. but the second we vew the pdf, it contains an extra white page. If i am not making sense, Please let me know anything I can provide for you to see.

**Comment by phuoc-ng** [comment]
I am not familiar with Electron, but can you let me know:
* Does the issue happen with the normal viewer (web-based viewer, not embedded within an Electron app)?
You can drag the document to https://react-pdf-viewer.dev, and click the _Print_ button.

* Can you create a minimum boilerplate (a Github repos, a Sandbox fiddle, etc), so I can try myself to reproduce the issue?

**Comment by sgil-carecloud** [comment]
@phuoc-ng I dont think the issue happens in a browser, and the reason we are using the library is because electrons embedded lib is having multiple bus to work. I will get a github repo running so I can show you this issue.

**Comment by sgil-carecloud** [comment]
@phuoc-ng here you go, https://github.com/sgil-carecloud/pdftest let me know if you have any trouble, npm install, npm run dev, and the pdf in question should pop up, hit print, save as pdf, and you will see an extra white page saves. thank you for responding quickly!

**Comment by aholland-carecloud** [comment]
@phuoc-ng @sgil-carecloud I was able to reproduce this bug in the current release `1.7.0` on the web. Sample Repo: https://github.com/aholland-carecloud/react-pdf-extra-page

However it seems that https://react-pdf-viewer.dev/ doesn't have the same issue, not sure why that is the case. 

I was able to debug enough to figure out that there is an extra div that is adding a tiny bit of margin causing an extra page. Not sure which div is the culprit though. The following css was enough to patch this bug, however I would love if we could figure out what is the root cause of this issue.

```
@media print {
  /* Remove the extra margin that is getting added while printing */
  @page {
    margin: 0;
  }
  /* Hide everything while printing */
  * {
    display: none;
  }
  /*
    Show only the content that we want to print, react-pdf-viewer's container div for the content contains `print-page-thumbnail`
    https://github.com/phuoc-ng/react-pdf-viewer/blob/v1.7.0/src/print/PageThumbnail.tsx#L71
  */
  body,
  html,
  div[class*='print-page-thumbnail'],
  div[class*='print-page-thumbnail'] > * {
    display: block;
  }
}
```

**Comment by vferreir42** [comment]
Same issue, in a web-based viewer, @aholland-carecloud fix doesn't work for me

**Comment by phuoc-ng** [comment]
@sgil-carecloud @aholland-carecloud 
Thanks for creating the repos for me. It's very helpful for me to reproduce the issue.

The issue was fixed in #332. I tested with web-based viewer and electron app.

_Demo_

Before (2 pages)

<img width="1156" alt="Screen Shot 2020-10-29 at 20 34 58" src="https://user-images.githubusercontent.com/57786711/97588407-08e5e000-1a2f-11eb-8bee-9b2af98d203d.png">

After (1 page)

<img width="1155" alt="Screen Shot 2020-10-29 at 20 38 33" src="https://user-images.githubusercontent.com/57786711/97588470-19965600-1a2f-11eb-8180-b634c52cde35.png">

The fix will be released in the next version, and I will update here when it comes officially.
